### PR TITLE
Overlap Changes

### PR DESCRIPTION
**Pull request by mhickle**
Overlap found with:
.navbar
.dropdown-menu
.alert
.active
.hidden
Changed to:
.wc-navbar
.wc-dropdown-menu
wc-.alert
.wc-active
.wc-hidden

### CODE REVIEW

File: src/chart/layout.js [hunk 1]
```diff
@@ -5,7 +5,7 @@
 export function layout() {
     var wrapper = this.wrap
         .append('div')
-        .attr('class', 'aeTable row-fluid')
```
Comment (pburnsdata) [inline src/chart/layout.js]: @samussiah I don't understand this update, so wanted to draw your attention to it

Comment (samussiah) [reply src/chart/layout.js]: Doesn't look like we need it, although it is a bootstrap class.  Go ahead and remove row-fluid, wc-row-fluid, and form-inline.  They're not referenced anywhere else.

Comment (pburnsdata) [reply src/chart/layout.js]: @samussiah Sounds good, removed those classes

**Comment by danedexF5** [comment]
@samussiah  @emmorris  I do not see any overlapping of columns on any of the three browsers. I compared them side by side with one having bootstrap toggled off and one with it on. There were some small css differences but nothing that was a big concern (coloring of Category names, Search bar in center of the page). When details are expanded and bootstrap is off they are sort of smushed together but not overlapped, basically, it doesn't look as nice. If you'd like me to test it in another way, I can do that, otherwise I think it's good.

**Comment by samussiah** [comment]
Nope, that'll do it!

**Comment by emmorris** [comment]
I agree that should be sufficient testing.  I looked them over a bit and saw the same types of differences.  

With Bootstrap on:
- Font style is slightly different
- Search box is in the far right corner instead of closer to the middle
- Category names are blue instead of black
- When you search the number of matches is in a green box with white text instead of just plain black text
- There is slightly less vertical spacing between boxes in the header section
- When you search on a term and clear the search, the category names that were highlighted as a search result are black text instead of blue
- The "Return to Summary View" button is larger and blue with white text instead of grey with black text
- There are horizontal lines between each subject record in the listing view

These are all minor and I confirmed that all of them except the vertical spacing and horizontal line ones are already in the latest version of the renderer in RhoMAP, so I don't think this CSS update negatively impacted anything.  We can discuss whether or not the other differences are worth addressing later.

**Pull request merged**